### DISCUSSION TITLE
What can be an estimated benchmark runtime and cost in GCP deployment

### DISCUSSION THREAD

**Discussion by Ritwaj-1** [openai/whisper#1267] [Q&A]
1000 hours of call data transcription monthly max, Around 100 hours weekly. I am using the large model in my transcription code. What configuration should I go for during deployment for the best results at not much of a cost?

**Comment by Jain-Archit** [comment]
Do you have a benchmark for this @Ritwaj-1 ? Also, how/what stack are you using to deploy whisper to production?
